roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -16,3 +16,23 @@ def is_active(cls, task_vars):
     @staticmethod
     def is_containerized(task_vars):
         return get_var(task_vars, "openshift", "common", "is_containerized")
+
+
+class NeedsYumMixin(object):
+    """Mixin for checks that are only active when 'yum' is available as the package manager on the host."""
+
+    @classmethod
+    def is_active(cls, task_vars):
+        return (
+            # This mixin is meant to be used with subclasses of OpenShiftCheck.
+            super(NeedsYumMixin, cls).is_active(task_vars) and
+            not cls.is_containerized(task_vars) and cls.has_yum_pkg_mgr(task_vars)
```

Comment:
@juanvallejo this mixin should do one thing -- it has nothing to do with 'containerized'.
We can remove this part of the condition here and remove the `is_containerized` method.

Note how we use the two mixins together to compose their effects:

```python
class PackageAvailability(NotContainerizedMixin, NeedsYumMixin, OpenShiftCheck):
   ...
```